### PR TITLE
feat: initial attempt at adding OpenZiti to edgex-go

### PR DESCRIPTION
**Pull request by dovholuknf**
WIP: I am only putting this PR up to illustrate how to configure the various services when running locally. There are changes here that likely need to be reverted/changed. This PR is for discussion on initial implemenation

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: cmd/core-command/res/configuration.yaml [hunk 1]
```diff
@@ -13,11 +13,20 @@ Service:
   Host: localhost
   Port: 59882
   StartupMsg: "This is the Core Command Microservice"
+  SecurityOptions:
+    Mode: "zerotrust"
+    OpenZitiController: "openziti:1280"
+    OpenZitiServiceName: "edgex.core-command"
```
Comment (lenny-intel) [inline cmd/core-command/res/configuration.yaml:19]: Put these in the `all-services` section of core-common-config-bootstrapper
and remove them from each service's private config

File: cmd/core-data/res/configuration.yaml [hunk 2]
```diff
@@ -9,16 +9,22 @@ Writable:
 #    Tags: # Contains the service level tags to be attached to all the service's metrics
     ##    Gateway="my-iot-gateway" # Tag must be added here or via Consul Env Override can only change existing value, not added new ones.
 Service:
-  Port: 59880
+  Port: 59810
```
Comment (lenny-intel) [inline cmd/core-data/res/configuration.yaml:12]: Why change the port number?

File: cmd/core-data/res/configuration.yaml [hunk 3]
```diff
@@ -9,16 +9,22 @@ Writable:
...
+    OpenZitiServiceName: "edgex.core-data"
 
 MessageBus:
   Optional:
     ClientId: "core-data"
 
 Database:
   Name: "coredata"
+  Host: "127.0.0.1"
+  Port: 6379
```
Comment (lenny-intel) [inline cmd/core-data/res/configuration.yaml:27]: This is already set in common config. 
When running from command line in hybrid mode use `-cp -d -o` flags

File: cmd/security-file-token-provider/res/token-config.json [hunk 4]
```diff
@@ -175,5 +175,8 @@
         }
       }
     }
+  },
+  "rules-engine": {
+    "edgex_use_defaults": true
```
Comment (lenny-intel) [inline cmd/security-file-token-provider/res/token-config.json:180]: Why is this needed? 
should be `app-rules-engine` if really needed

File: cmd/support-scheduler/res/configuration.yaml [hunk 5]
```diff
@@ -2,9 +2,14 @@ ScheduleIntervalTime: 500
 Writable:
     LogLevel: INFO
 Service:
-    Host: localhost
-    Port: 59861
+    Host: support-scheduler.edgex.ziti
+    Port: 80
```
Comment (lenny-intel) [inline cmd/support-scheduler/res/configuration.yaml:6]: why port change?

File: internal/support/scheduler/application/scheduler/manager.go [hunk 6]
```diff
@@ -6,6 +6,7 @@
 package scheduler
 
 import (
+	clientInterfaces "github.com/edgexfoundry/go-mod-core-contracts/v3/clients/interfaces"
```
Comment (lenny-intel) [inline internal/support/scheduler/application/scheduler/manager.go:9]: restore order of imports

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: cmd/security-file-token-provider/res/token-config.json (hunk 4 above)
Comment (dovholuknf) [reply cmd/security-file-token-provider/res/token-config.json:180]: my understanding is that app-rules-engine and ekuiper are not the same, right? I used the name of the service as found in the docker-file which is `rules-engine:` for ekuiper. it also follows along with the REST requests which are performed by the UI which issues a GET to `http://localhost:4000/rules-engine/ping`
